cfme/base/ui.py

```diff
@@ -1257,7 +1254,6 @@ def step(self):
 
 
 class AutomateImportExportBaseView(BaseLoggedInPage):
-    flash = FlashMessages('div.import-flash-message')
```

Comment:
This flash div is actually unique from the flash widget in BaseLoggedInPage, and should be kept.

I think it actually needs to be a unique name, like `import_flash`, because the page has the regular flash message div and this import div.  Trying to import an invalid file puts a message in the import-flash div, not the standard flash div.